jetbrains-core/src/software/aws/toolkits/jetbrains/core/notification/JetBrainsMinimumVersionChange.kt

```diff
@@ -29,7 +30,7 @@ class JetBrainsMinimumVersionChange : NoticeType {
     override fun isNotificationRequired(): Boolean {
         val appInfo = ApplicationInfo.getInstance()
         val majorVersion = appInfo.majorVersion.toIntOrNull()
-        val minorVersion = appInfo.minorVersion.toIntOrNull()
+        val minorVersion = appInfo.minorVersion.toFloatOrNull()
```

Comment:
Strange. Was that a JB/legacy/historical choice , or why was a float used?

Reply:
It's actually a string "2.2" which obviously didn't parse as an `Int`

Reply:
the `toFloatOrNull` is a kotlin built-in extension function on `String`

Reply:
In a version string x.y.z., "y" is the minor version and "z" is the "patch", and both could be integers then.